### DISCUSSION TITLE
Hi, can you help me? I try to set a agentGPT by colab, but there is a problem

### DISCUSSION THREAD

**Discussion by parkshinwan** [reworkd/AgentGPT#1031] [Q&A]
I am setting agentGPT using colab. I'm following the guide, but when I run ./setup.bat from colab as !./setup.bat. A message like the following will appear:

audited 154 packages in 1.005s

30 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

> agentgpt-cli@1.0.0 start /content/AgentGPT/cli
> node src/index.js

 █████╗  ██████╗ ███████╗███╗   ██╗████████╗ ██████╗ ██████╗ ████████╗
██╔══██╗██╔════╝ ██╔════╝████╗  ██║╚══██╔══╝██╔════╝ ██╔══██╗╚══██╔══╝
███████║██║  ███╗█████╗  ██╔██╗ ██║   ██║   ██║  ███╗██████╔╝   ██║   
██╔══██║██║   ██║██╔══╝  ██║╚██╗██║   ██║   ██║   ██║██╔═══╝    ██║   
██║  ██║╚██████╔╝███████╗██║ ╚████║   ██║   ╚██████╔╝██║        ██║   
╚═╝  ╚═╝ ╚═════╝ ╚══════╝╚═╝  ╚═══╝   ╚═╝    ╚═════╝ ╚═╝        ╚═╝   
                                                                      
Welcome to the AgentGPT CLI! This CLI will generate the required .env files.
Copies of the generated envs will be created in `./next/.env` and `./platform/.env`.

? How will you be running AgentGPT? (Use arrow keys)
❯ 🐋 Docker-compose (Recommended) 
  💪 Manual (Not recommended) 

That's all! 
so, I can't see instructions from the script - can add the appropriate API keys!! 
Even when I go to the site(http://localhost:3000/ ), I get a message that it is denied by localhost.

What I should do? I eagerly want to use agentgpt by setting myself. thank you

**Comment by jasangill1** [comment]
Hello, @parkshinwan not sure if you can set up in collab. Have you tried installing with docker ??
